### ISSUE TITLE
TypeError: 'NoneType' object is not subscriptable on find query

### ISSUE THREAD

**Issue by razzaghi** [redis/redis-om-python#146]
Hi Dears i got error on find query with below information:

class Customer(HashModel):
    first_name: str
    last_name: str = Field(index=True)

    class Meta:
        database = redis

Customer.find(Customer.last_name == "Brookins").all()

Error is: 
TypeError: 'NoneType' object is not subscriptable

**Comment by msarm** [comment]
I'm facing the same issue. Any guidance from anyone will help.

**Comment by talsht** [comment]
I think this has to do with the redis deployment you are using.

as the readme mentions : 
"Now, if we use this model with a Redis deployment that has the RediSearch module installed, we can run queries like the following."

I got the same exception when trying to run with a regular redis docker (image redis:latest).
When trying to run with the image in the instructions (redis/redis-stack:latest) I did manage to use the find query without any exception. Hope this helps.
